firefox-ios/Client/Frontend/TrackingProtection/BlockedTrackersHelpers/BlockedTrackerCell.swift

```diff
@@ -0,0 +1,96 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Common
+import Shared
+
+class BlockedTrackerCell: UITableViewCell,
+                          ReusableCell {
+    private struct UX {
+        static let imageMargins: CGFloat = 10
+        static let textVerticalDistance: CGFloat = 11
+        static let cornerRadius: CGFloat = 5
+    }
+
+    private let trackerImageView: UIImageView = .build { image in
+        image.contentMode = .scaleAspectFit
+        image.clipsToBounds = true
+        image.layer.masksToBounds = true
+        image.layer.cornerRadius = UX.cornerRadius
+    }
+
+    private let trackerLabel: UILabel = .build { label in
+        label.font = FXFontStyles.Regular.subheadline.scaledFont()
+        label.numberOfLines = 0
+        label.adjustsFontForContentSizeCategory = true
+        label.textAlignment = .left
+    }
+
+    public let dividerView: UIView = .build { _ in }
+    private var trackerImageViewHeightConstraint: NSLayoutConstraint?
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupView()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupView() {
+        contentView.addSubview(trackerImageView)
+        contentView.addSubview(trackerLabel)
+        contentView.addSubview(dividerView)
+
+        trackerImageViewHeightConstraint = trackerImageView.heightAnchor.constraint(equalToConstant: TPMenuUX.UX.iconSize)
+        NSLayoutConstraint.activate([
+            trackerImageView.centerYAnchor.constraint(equalTo: contentView.centerYAnchor),
+            trackerImageView.leadingAnchor.constraint(
+                equalTo: contentView.leadingAnchor,
+                constant: TPMenuUX.UX.horizontalMargin
+            ),
+            trackerImageView.heightAnchor.constraint(equalTo: trackerImageView.widthAnchor),
+            trackerImageViewHeightConstraint!,
```

Comment:
remove forced unwrapped?

Reply:
I asked on the team for their stance as well, let me know what you think

Reply:
I also prefer not to use forced unwrapping, but since `trackerImageViewHeightConstraint` is initiated just above, we should be okay. To ensure everything's safe and to remove the scary `!`, I'll make it a lazy variable instead.